docs/users.rst

```diff
@@ -428,8 +412,6 @@ modular_koji_tags
 must have a non-empty list of modules. The "packages" key, and only the "packages"
 key, can have an empty list.**
 
-**The "packages", "modules" and "pulp_repos" keys can be used mutually.**
```

Comment:
Should we keep the information that the package and the modules keys can be used mutually?

Reply:
We need to keep this part altogether.

The `pulp_repos` option is about RPMs stored in Pulp, not container images.